edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -59,6 +61,21 @@ def get_jwt_claim_mergeable_attributes(self):
         return get_setting('JWT_PAYLOAD_MERGEABLE_USER_ATTRIBUTES')
 
     def authenticate(self, request):
+        # Add monitoring to help resolve issues with JWT cookies.
+        # Note: In the very exceptional case that a JWT authorization header is also used
+        #   and takes precedence over the JWT cookies, this custom attribute
+        #   could be slightly misleading.
+        has_jwt_cookie = jwt_cookie_name() in request.COOKIES
+        set_custom_attribute('jwt_auth_has_jwt_cookie', has_jwt_cookie)
+
+        is_forgiving_jwt_cookies_enabled = get_setting(ENABLE_FORGIVING_JWT_COOKIES)
+        if is_forgiving_jwt_cookies_enabled:
+            set_custom_attribute('jwt_auth_forgiving_jwt_cookies', True)
```

Comment:
1. I feel like we should add `enabled` to the name, if we keep this attribute, to clarify this is just about the toggle. This is a long name though.
2. I’m wondering if it would be better to have an additional or replacement attribute like jwt_auth_failure_forgiven, which would be True when this new functionality actually has an effect.

Reply:
I'll make these updates, today or tomorrow.

Reply:
I'm taking care of this. I'll commit changes next week when I return.

Reply:
See https://github.com/openedx/edx-drf-extensions/pull/197#pullrequestreview-1572779330 for changes. Marking as resolved.